tqa-web-app/src/app/dtos/v2/admin-auth.dto.v2.ts

```diff
@@ -0,0 +1,22 @@
+//saving token in the ngrx state
+export interface AdminAuthResponseV2{
+    email: string;
+    token: string;
+    expiresIn: string;
+
+}
+
+export class AdminV2{
+    constructor(
+        private email: string,
+        private token: string,
+        private expirationDate: Date
+      ) {}
+    
+    get adminEmail(){
+        return this.email;
+    }
+    get expireDate(){
+        return this.expirationDate;
+    }
+}
```

Comment:
Not positive what this class is for?  You should be able to create an instance of AdminAuthResponseV2 just as easily and that would be the preferred pattern?

Reply:
adminauthresponseV2 take expiresIn in string, adminV2 turn the expiresIn into expirationDate: Date